packages/bot-skeleton/src/scratch/hooks/css.js

```diff
@@ -64,7 +64,7 @@ Blockly.Css.CONTENT = [
     '}',
 
     '.injectionDiv {',
-    'height: 100%;',
+    'height: calc(100vh - 11rem);',
```

Comment:
@rupato-deriv we have this change in my PR

Reply:
done ✅